arbitrum-docs/arbos/l1-to-l2-messaging.mdx

```diff
@@ -1,21 +1,125 @@
+---
+title: "L1-to-L2 messaging on Arbitrum: A conceptual overview"
+sidebar_label: "L1-to-L2 messaging"
+description: "Learn how Arbitrum's Layer 2 (L2) chains communicate with Ethereum's Layer 1 (L1) chain using retryable tickets."
+author: mahsamoosavi
+reader_audience: new developers who want to build Arbitrum dApps
+content_type: concept
+---
+
 import { AddressAliasHelper } from '@site/src/components/AddressAliasHelper'
 import { MermaidWithHtml, Nodes, Node, Connection, NodeDescriptions, NodeDescription } from '/src/components/MermaidWithHtml/MermaidWithHtml';
 
 
-# L1 To L2 Messaging
+<a data-quicklook-from='arbitrum-one'>Arbitrum One</a> and <a data-quicklook-from='arbitrum-nova'>Arbitrum Nova</a> are Layer 2 (L2) chains that continuously exchange information with Ethereum's Layer 1 (L1) chain. This concept doc explains when and how information is sent from Ethereum's L1 chain to Arbitrum's L2 chains.
+
+<br /> <br />
+
+Cross-layer messaging is a complex topic, so this is a lengthy concept doc that progressively discloses technical complexity. Here's what we'll cover:
+
+  1. **[Why this information is important](#1-why-this-information-is-important)**.
+  2. **[The conceptual primitives](#2-the-conceptual-primitives)** that work together to enable L1-to-L2 messaging on Arbitrum.
+  3. **[Elaborating on the conceptual primitives](#3-elaborating-on-the-conceptual-primitives)** by exploring how each of the conceptual primitives relate to one another in the context of cross-layer messaging.
+  4. **[Learning by example](#4-learning-by-example)** with a minimal smart contract that exposes two functions. We'll compare the contract's behavior across two across deployment scenarios to test our understanding of the conceptual primitives.
+
+
+This document doesn't assume any prior knowledge beyond familiarity with the following terms:
+
+ 1. **Layers**: We'll focus on the first two: L1 (Ethereum's chains) and L2 (Arbitrum's chains). Both layers host multiple chains; each chain is supported by its own decentralized network of nodes.
```

Comment:
> Both layers host multiple chains

regarding L2s hosting multiple chains, is this referring to L3s? If so, might be a bit confusing to get into